src/handlers/users/config.yml

```diff
@@ -0,0 +1,20 @@
+users:
+  handler: src/handlers/users/list.list
+  events:
+    - http: true
+      x-azure-settings:
+        authLevel : anonymous
+    - http: true
+      x-azure-settings:
+        direction: out
+        name: res
+usersadd:
```

Comment:
I would like to make this as `post` instead of `get` and I'd like to use the path `/users`. Anyone can help me with that?
```

Reply:
this is done in the function.json, there should be a function.json per function handler